src/game/default/g_physics.c

```diff
@@ -23,6 +23,33 @@
 #include "bg_pmove.h"
 
 /**
+ * @brief Spawn a liquid ripple between a start and end point, setting pos1 to
+ * NULL will not test a trace
+ */
+void G_LiquidRipple(g_entity_t *ent, const vec3_t pos1, const vec3_t pos2, const vec_t size) {
+	vec3_t end;
+
+	_Bool doadd = true;
+
+	if (pos1 != NULL) {
+		cm_trace_t tr = gi.Trace(pos1, pos2, NULL, NULL, ent, MASK_LIQUID);
```

Comment:
`tr` can be `const`. Also, I'd prefer a `return` instead of the `doadd` business.